robust/signomial_simple_wing/models.py

```diff
@@ -1,101 +1,9 @@
-    C_D = Variable("C_D", "-", "Drag coefficient")
-    C_L = Variable("C_L", "-", "Lift coefficent of wing")
-    C_f = Variable("C_f", "-", "skin friction coefficient")
-    W_f = Variable("W_f", "N", "fuel weight")
-    V_f = Variable("V_f", "m^3", "fuel volume")
-    T_flight = Variable("T_{flight}", "hr", "flight time")
-
-    # Free variables (fixed for performance eval.)
-    A = Variable("A", "-", "aspect ratio", fix=True)
-    S = Variable("S", "m^2", "total wing area", fix=True)
-    W_w = Variable("W_w", "N", "wing weight")
-    W_w_strc = Variable('W_w_strc', 'N', 'wing structural weight', fix=True)
-    W_w_surf = Variable('W_w_surf', 'N', 'wing skin weight', fix=True)
-    V_f_wing = Variable("V_f_wing", 'm^3', 'fuel volume in the wing', fix=True)
-    V_f_fuse = Variable('V_f_fuse', 'm^3', 'fuel volume in the fuselage', fix=True)
-    V_f_avail = Variable("V_{f_{avail}}", "m^3", "fuel volume available")
-    constraints = []
-
-    # Drag model
-    C_D_fuse = CDA0 / S
-    C_D_wpar = k * C_f * S_wetratio
-    C_D_ind = C_L ** 2 / (np.pi * A * e)
-    constraints += [C_D >= C_D_fuse * toz + C_D_wpar / toz + C_D_ind * toz]
-
-    with SignomialsEnabled():
-        # Wing weight model
-        # NOTE: This is a signomial constraint that has been GPified. Could revert back to signomial?
-        constraints += [W_w >= W_w_surf + W_w_strc,
-                        # W_w_strc >= W_W_coeff1 * (N_ult * A ** 1.5 * ((W_0+V_f_fuse*g*rho_f) * W * S) ** 0.5) / tau, #[GP]
-                        W_w_strc ** 2. >= W_W_coeff1 ** 2. * (
-                            N_ult ** 2. * A ** 3. * ((W_0 + V_f_fuse * g * rho_f) * W * S)) / tau ** 2.,
-                        W_w_surf >= W_W_coeff2 * S]
-
-        # and the rest of the models
-        constraints += [LoD == C_L / C_D,
-                        D >= 0.5 * rho * S * C_D * V ** 2,
-                        Re <= (rho / mu) * V * (S / A) ** 0.5,
-                        C_f >= 0.074 / Re ** 0.2,
-                        T_flight >= Range / V,
-                        W_0 + W_w + 0.5 * W_f <= 0.5 * rho * S * C_L * V ** 2,
-                        W <= 0.5 * rho * S * C_Lmax * V_min ** 2,
-                        W >= W_0 + W_w + W_f,
-                        V_f == W_f / g / rho_f,
-                        V_f_avail <= V_f_wing + V_f_fuse,  # [SP]
-                        V_f_wing ** 2 <= 0.0009 * S ** 3 / A * tau ** 2,  # linear with b and tau, quadratic with chord!
-                        V_f_fuse <= 10 * units('m') * CDA0,
-                        V_f_avail >= V_f,
-                        W_f >= TSFC * T_flight * D]
-
-    # return Model(W_f/LoD, constraints)
-    return Model(D, constraints)
-    # return Model(W_f, constraints)
-    # return Model(W,constraints)
-    # return Model(W_f*T_flight,constraints)
-    # return Model(W_f + 1*T_flight*units('N/min'),constraints)
-
+from gplibrary.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission
```

Comment:
Maybe this should be from gplibrary.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission by gpkitmodels.SP.SimPleAC.SimPleAC_mission import SimPleAC, Mission